### ISSUE TITLE
Using Mutiple GPUs

### ISSUE THREAD

**Issue by csBob123** [TUI-NICR/ESANet#3]
Hi, I found only the GPU 0 will be used in training, which leads to the low efficiency in training.

**Comment by mona0809** [comment]
Yes, we only trained on one GPU. The code can probably be adapted to Multi GPU training by following official tutorials.
